src/core/DataFile.cpp

```diff
@@ -868,10 +869,21 @@ void DataFile::upgrade_1_1_91()
 	for( int i = 0; !list.item( i ).isNull(); ++i )
 	{
 		QDomElement el = list.item( i ).toElement();
-		if ( el.attribute( "name" ) == "plugin" && el.attribute( "value" ) == "vocoder-lmms" ) {
+		if( el.attribute( "name" ) == "plugin" && el.attribute( "value" ) == "vocoder-lmms" ) {
 			el.setAttribute( "value", "vocoder" );
 		}
 	}
+
+	list = elementsByTagName( "crossoevereqcontrols" );
+	for( int i = 0; !list.item( i ).isNull(); ++i )
+	{
+		QDomElement el = list.item( i ).toElement();
+		// invert the mute LEDs
+		for( int d = 1; d <= 4; ++d ){
```

Comment:
@tresf I tried doing it with a while loop but `el.hasAttribute( a )` kept returning true all the time. Anyways, I'll let it stay as is, but I've changed the variable names to your comments.